### ISSUE TITLE
1.22-alpha - java.lang.RuntimeException: Please provide a valid Mongo connection uri.

### ISSUE THREAD

**Issue by bappr** [prisma/prisma#3582]
**Describe the bug**
```
Exception in thread "main" java.lang.RuntimeException: Unable to load Prisma config: java.lang.RuntimeException: Please provide a valid Mongo connection uri.
	at scala.sys.package$.error(package.scala:27)
	at com.prisma.config.ConfigLoader$.load(ConfigLoader.scala:40)
	at com.prisma.local.PrismaLocalDependencies.<init>(PrismaLocalDependencies.scala:37)
	at com.prisma.local.PrismaLocalMain$.delayedEndpoint$com$prisma$local$PrismaLocalMain$1(PrismaLocalMain.scala:11)
	at com.prisma.local.PrismaLocalMain$delayedInit$body.apply(PrismaLocalMain.scala:8)
	at scala.Function0.apply$mcV$sp(Function0.scala:34)
	at scala.Function0.apply$mcV$sp$(Function0.scala:34)
	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
	at scala.App.$anonfun$main$1$adapted(App.scala:76)
	at scala.collection.immutable.List.foreach(List.scala:389)
	at scala.App.main(App.scala:76)
	at scala.App.main$(App.scala:74)
	at com.prisma.local.PrismaLocalMain$.main(PrismaLocalMain.scala:8)
	at com.prisma.local.PrismaLocalMain.main(PrismaLocalMain.scala)
```

**To Reproduce**
Use prisma docker image `1.22-alpha`

Can you guys patch the docker images instead of only overriding the minor version?

**Comment by mavilein** [comment]
@bappr : Thanks a lot for reaching out to us and reporting that bug! 🙏 I have 2 questions:
1. What config were you using that caused that error? 
2. Can you elaborate what you mean with your last question?

**Comment by iagomelanias** [comment]
Related to the PR https://github.com/prisma/prisma/pull/3565.
**TLDR**: authentication parameters (host, port, user, etc) were replaced by uri, like most mongo clients.

> Can you guys patch the docker images instead of only overriding the minor version?

Yep! Incremental alpha/beta releases would be very helpful, so projects in progress that relies on upcoming features won't be blocked. 😅

**Comment by mavilein** [comment]
@bappr : Ah now i get it. Our `alpha` channel is intentionally bleeding edge and we do not want to pollute our docker repository with enormous amounts of alpha builds. 
Yesterday the mongo connector was merged to `beta` and you can from now on rely on this more stable tag. In 2 weeks the mongo connector will hit the stable channel and this problem will be gone completely :-)

**Comment by bappr** [comment]
Awesome! Thanks @mavilein
